packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -89,7 +89,7 @@ export async function getBalance<ReturnFormat extends DataFormat>(
 	blockNumber: BlockNumberOrTag = web3Context.defaultBlock,
 	returnFormat: ReturnFormat,
 ) {
-	const blockNumberFormatted = isBlockTag(blockNumber)
+	const blockNumberFormatted = isBlockTag(blockNumber as string)
```

Comment:
@nazarhussain I had to add these back because I changed `BlockNumberOrTag` from `Uint | BlockTag` to `Numbers | BlockTag`